db.go

```diff
@@ -2,13 +2,17 @@ package arcticdb
 
 import (
 	"fmt"
+	"path"
 	"sync"
 	"time"
 
 	"github.com/go-kit/log"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/thanos-io/objstore"
 	"go.uber.org/atomic"
+)
 
+import (
```

Comment:
I'm so sorry, I phrased myself poorly, the way the linter wants it is to have one import statement where stdlib, external and internal imports are blocks that are separated just by newlines, so what it wants here is:

```
import (
	"fmt"
	"path"
	"sync"
	"time"

	"github.com/go-kit/log"
	"github.com/prometheus/client_golang/prometheus"
	"github.com/thanos-io/objstore"
	"go.uber.org/atomic"

	"github.com/polarsignals/arcticdb/query/logicalplan"
)
```